mmiwg2s-backend/init-mongo.js

```diff
@@ -0,0 +1,12 @@
+db.createUser(
+  {
+    user: "test",
+    pwd: "secret",
```

Comment:
Will these remain the same? Is this a security issue?

Reply:
Not using this. I removed it.